src/pages/Claim/StartNewClaimSection.tsx

```diff
@@ -14,6 +15,13 @@ type Props = {
 export const StartNewClaimSection: React.FC<Props> = ({ protocol }) => {
   const [{ data: connectedAccount }] = useAccount()
   const [isCreating, setIsCreating] = useState(false)
+  const [canStartNewClaim, setCanStartNewClaim] = useState(false)
+
+  useEffect(() => {
+    setCanStartNewClaim(
+      !!connectedAccount?.address && ethers.utils.getAddress(connectedAccount.address) === protocol.agent
+    )
+  }, [connectedAccount?.address])
```

Comment:
Applies also to `NewClaimModal.tsx line 48` and `ClaimStatusAction.tsx line 67`

`protocol.agent` hook dep.

Any reason for using `useState` instead of a simple `useMemo`?